service/src/test/java/bio/terra/workspace/service/resource/controlled/ControlledAiNotebookInstanceResourceTest.java

```diff
@@ -58,7 +58,7 @@ public void testFlightMapSerialization() {
     serializeMap.put("resource", resource);
 
     FlightMap deserializedMap = new FlightMap();
-    deserializedMap.fromJson(serializeMap.toJson());
```

Comment:
I think this meets the spirit of what the test was trying to accomplish...

Reply:
I think just testing `.put` and `.get` is sufficient now actually. The point of this test is to verify that the `ControlledAiNotebookInstanceResource` can be serdes'd correctly by flightmap. Since flight map serialization happens at `put` and `get` now (instead of previously at `toJson -> fromJson`), we don't actually need the raw conversions.
But maybe that's not a part of the flightmap contract. Then again, I'm not sure that this use of `put -> getRaw -> putRaw -> get` is either.

Reply:
OK, my thinking was that this was explicitly testing that Stairway's JSON representation serialized and deserialized, but since the intent is to just make sure `FlightMap` can handle it, `put/get` make more sense (and read better).